Octokit/Helpers/AcceptHeaders.cs

```diff
@@ -2,6 +2,14 @@
 {
     public static class AcceptHeaders
     {
+        public const string DefaultJson = "application/vnd.github.quicksilver-preview+json; charset=utf-8, application/vnd.github.v3+json; charset=utf-8";
+
+        public const string DefaultHtml = "application/vnd.github.html";
+
+        public const string ReleaseApi = "application/vnd.github.v3";
```

Comment:
Let's name this something like `StableVersion` as its the replacement for the releases API now that's out of preview